Examples/ImageRegistrationMethod4/Documentation.rst

```diff
@@ -1,11 +1,86 @@
 Image Registration Method 4
 ===========================
 
-.. include:: registrationExamplePrefix.rst
-
 Overview
 --------
 
+.. include:: registrationExamplePrefix.rst
+
+Example Run
+-----------
+
+Running the Python code with the following inputs:
+
+``main('BrainProtonDensitySliceBorder20.png', 'BrainProtonDensitySliceShifted13x17y.png', 'displaceMeth4.hdf5')``
```

Comment:
See comment with respect to `Documentation.rst` for method 2.

Reply:
Agreed. These made more sense at the beginning of my task, but now they are just confusing. Replacing them with a generic version makes more sense. Ideally, there would be a whole usage section, but the code was in too much flux for that to make sense.